Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM kbase/kbase:sdkbase.latest
-MAINTAINER KBase Developer
+FROM kbase/sdkbase2:latest
+MAINTAINER 3rdParty Developer
```

Comment:
This should really be an email address for the developer. For example, Mikaela's email